src/main/java/graphql/util/Anonymizer.java

```diff
@@ -741,17 +741,6 @@ public TraversalControl visitDirective(Directive directive, TraverserContext<Nod
                 return changeNode(context, directive.transform(builder -> builder.name(newName)));
             }
 
-//            @Override
-//            public TraversalControl visitStringValue(StringValue node, TraverserContext<Node> context) {
-//                return changeNode(context, node.transform(builder -> builder.value("stringValue" + stringValueCounter.getAndIncrement())));
-//            }
-//
-//            @Override
-//            public TraversalControl visitIntValue(IntValue node, TraverserContext<Node> context) {
-//                return changeNode(context, node.transform(builder -> builder.value(BigInteger.valueOf(intValueCounter.getAndIncrement()))));
-//            }
-//
-
```

Comment:
Was this meant?

Reply:
deleting commented out code .. not directly related to this PR